### ISSUE TITLE
Ability to bind an application running in CRW workspace to a managed service

### ISSUE THREAD

**Issue by jfdenise** [redhat-developer/app-services-cli#1341]
### Feature or problem description

Today the rhoas CLI bind command requires a service and an app to inject the secret in the app deployment.

I am in a context where I want to bind a deployment running inside a CRW workspace container to a service (eg: kafka).

I can't target the CRW workspace deployment, the CRW POD will not get re-started (that is the CRW behavior).

### Proposed solution

* Add the ability to create the secret that would be bound without providing an app. The secret would then be explicitly mounted 

rhoas cluster create-secret --service-name kafka

* Offer an out of the box solution to bind to CRW workspaces.

The [doc](https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces/2.13/html/end-user_guide/developer-workspaces_crw#mounting-a-secret-as-a-file-or-an-environment-variable-into-a-workspace-container_crw)  explains how secrets are to be formatted for CRW to automatically mount them.

Something like: rhoas cluster bind --service-name kafka --crw-workspace

The secret would then be created in the user project with the CRW expected labels. Secret would then be automatically mounted when the workspace starts.

**Comment by wtrocki** [comment]
@jfdenise This fix is on our roadmap.
We going to build secret in cli and give developer ability to apply it the best way possible.

**Comment by maxandersen** [comment]
@wtrocki would these be mounted in same way as ServiceBinding spec ? that would be interesting imo .

**Comment by wtrocki** [comment]
Yes. Idea is to move from CLI creating resources using API to CLI creating files that you can include with your Quarkus app deployment, helm etc. Shared internal doc with you
